src/main/java/com/hubspot/jinjava/lib/tag/ForTag.java

```diff
@@ -159,6 +162,14 @@ public String interpret(TagNode tagNode, JinjavaInterpreter interpreter) {
                 entryVal = entry.getValue();
               }
 
+              interpreter.getContext().put(loopVar, entryVal);
+            } else if (List.class.isAssignableFrom(val.getClass())) {
+              List<Object> entries = ((PyList) val).toList();
+              String entryVal = null;
+              // safety check for size
+              if (entries.size() >= loopVarIndex) {
+                entryVal = Objects.toString(entries.get(loopVarIndex));
```

Comment:
can you add a test for this case?

Reply:
This is a safety check only but I've added tests where the `jinjava.render` is expected to throw an exception when there are lesses values to unpack. :+1: